conda_build/cli/main_debug.py

```diff
@@ -43,39 +42,37 @@ def parse_args(args):
                          "The top-level recipe can be specified by passing 'TOPLEVEL' here"))
     p.add_argument("-a", "--activate-string-only", action="store_true",
                    help="Output only the string to the used generated activation script.  Use this for creating envs in scripted "
-                   "environments.")
+                        "environments.")
 
     # cut out some args from render that don't make sense here
     #    https://stackoverflow.com/a/32809642/1170370
     p._handle_conflict_resolve(None, [('--output', [_ for _ in p._actions if _.option_strings == ['--output']][0])])
     p._handle_conflict_resolve(None, [('--bootstrap', [_ for _ in p._actions if _.option_strings == ['--bootstrap']][0])])
     p._handle_conflict_resolve(None, [('--old-build-string', [_ for _ in p._actions if
                                                               _.option_strings == ['--old-build-string']][0])])
-    args = p.parse_args(args)
-    return p, args
+    return p
 
 
-def execute(args):
-    p, _args = parse_args(args)
+def execute():
+    parser = get_parser()
+    args = parser.parse_args()
     test = True
```

Comment:
Looks like `test` no longer serves a purpose and can be removed (see `if test` clauses on lines 70 & 76 below).